applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -640,7 +640,9 @@ static void audio_datapath_i2s_blk_complete(uint32_t frame_start_ts, uint32_t *r
 		size_t size;
 
 		if (ret != prev_ret) {
+#if (CONFIG_STREAM_BIDIRECTIONAL)
```

Comment:
Tried to implement this, but the error " Caught system error - reason 0. Entering infinite loop".  Seems like the program crashes while in " audio_i2s_set_next_buf", but could not find the reason for this.

Reply:
Also, should I change the name of the branches I have created?

Reply:
We discussed this, and Khuong will do some investigation why it crashes.
No need to change the name of the branches you already have created

Reply:
The problem seems to be that when starting I2S the RX was always configured to be on.
Added necessary "#if" conditions to prevent this, and it should work now